GameEngine/GameState.cs

```diff
@@ -38,10 +38,10 @@ public void TakeTurn()
             }
             else
             {
-                var cardToPlay = Player.SelectCardToPlay(Board);
-                Console.WriteLine("Player chose to play a {0} card", cardToPlay);
-                Board.Move(cardToPlay);
-                Player.Discard(cardToPlay);
+                var play = Player.FormulatePlay(Board);
+                Console.WriteLine("Player chose to play a {0} card on an owl at position {1}", play.Card, play.Position);
```

Comment:
💡 It feels like `Play` should have its own `ToString` that is used here

Reply:
Easy enough. Done.